docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Mbed_USB_WAV_Audio_Player.md

```diff
@@ -16,39 +20,52 @@ This tutorial uses the USBAudio class of the USB library to send audio data from
 
 - Mbed Enabled board.
 - SD card.
-- SD card shield (if board does not already have an SD card port).
+- SD card shield (if the board does not already have an SD card port).
 - USB cable.
 
+[Please install Mbed CLI to complete the tutorial](../tools/installation-and-setup.html).
+
 ## Setup
 
 The following steps demonstrate the setup and use of the Mbed WAV audio player:
 
 1. Create a new Mbed project:
-   
+
    `mbed new <project name>`
-   
-1. Add the [sd-driver](https://github.com/ARMmbed/sd-driver) and [AudioPlayer](https://github.com/c1728p9/AudioPlayer) libraries into the new Mbed project directory: 
-   
+
+1. Add the [sd-driver](https://github.com/ARMmbed/sd-driver) and [AudioPlayer](https://github.com/c1728p9/AudioPlayer) libraries to the new Mbed project directory:
+
    `mbed add https://github.com/ARMmbed/sd-driver`    
    `mbed add https://github.com/c1728p9/AudioPlayer`
-   
-1. Copy and paste the below example code into `main.cpp`.
-1. Make sure the SPI pins for the SDBlockDevice object are updated and correct for your board. For example, in the example below, line 10 sets up SPI for the SDBlockDevice for the NXP K64F Mbed board.
-1. Load WAV file(s) onto SD card. The example below uses a public domain WAV file called "Bach-minuet-in-g.wav" (attached below for download) that is inside a "songs" directory on the SD card. It is important that the WAV file be PCM signed 16-bit little endian, or else it will not play becaues USBAudio does not support any other WAV formats. The WAV file can have any sampling rate and can have any number of channels. You can use [Online-convert](https://audio.online-convert.com/convert-to-wav) to achieve WAV files with different formats from any source audio file. The file "Bach-minuet-in-g.wav" is already in the correct format.
-   
+
+1. Copy and paste the example code below into `main.cpp`.
+1. Make sure the SPI pins for the SDBlockDevice object are correct for your board. For example, in the code below, line 10 sets up SPI for the SDBlockDevice for the NXP-K64F Mbed board.
+1. Load a WAV file onto the SD card. The example below uses a public domain WAV file called ["Bach-minuet-in-g.wav"](https://github.com/mrcoulter45/mbed-os-5-docs/raw/Mbed_USB_WAV_Audio_Player_Tutorial/docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Bach-minuet-in-g.wav) that is inside a "songs" directory on the SD card<!--no it isn't; you need to download it and put it on the SD card-->.
+
```

Comment:
in the `songs` directory (or even folder?)

Reply:
What I meant was that, in the actual main code at the bottom, the line ```if (file.open(&fs, "songs/Bach-minuet-in-g.wav") != 0) {``` is opening a file called Bach-minuet-in-g.wav that is located  in a file called songs that is located on the sd card, the user could just put the WAV file on a blank sd card if they wanted to. So maybe this step could be re-written more clearly like:
"1. Load a WAV file onto the SD card. This example uses a public domain WAV file called ["Bach-minuet-in-g.wav"](https://github.com/mrcoulter45/mbed-os-5-docs/raw/Mbed_USB_WAV_Audio_Player_Tutorial/docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Bach-minuet-in-g.wav). The example code below utilizes this WAV file by calling ```file.open(&fs, "songs/Bach-minuet-in-g.wav")```, which means that the WAV file is inside of a "songs" directory located on the SD card"

Reply:
Also, I believe the "https://github.com/mrcoulter45/mbed-os-5-docs/raw/Mbed_USB_WAV_Audio_Player_Tutorial/docs/tutorials/using_apis/Mbed_USB_WAV_Audio_Player/Bach-minuet-in-g.wav" link needs to be updated.